### PR TITLE
ISEA_STATIC -> static

### PR DESCRIPTION
**Pull request by schwehr**
As this is a part of PROJ, having these functions possibly not static
is confusing.  Time to declare this is just a part of PROJ.

### CODE REVIEW

**Comment by schwehr** [comment]
so much fun...
```
--2018-05-13 03:16:15--  (try:20)  http://download.osgeo.org/proj/proj-datumgrid-1.7.zip
Connecting to download.osgeo.org|140.211.15.132|:80... connected.
HTTP request sent, awaiting response... Read error (Connection reset by peer) in headers.
Giving up.
```

**Comment by kbevers** [comment]
> so much fun...

Yeah. I guess the server has gone home for the weekend :-) SAC team is trying to sort things out:

https://lists.osgeo.org/pipermail/sac/2018-May/010024.html

File: src/PJ_isea.c [hunk 1]
```diff
@@ -309,8 +291,7 @@ sph_azimuth(double f_lon, double f_lat, double t_lon, double t_lat)
 #endif
 
 /* coord needs to be in radians */
-ISEA_STATIC
-int
+static int
```
Comment (kbevers) [inline src/PJ_isea.c]: A small style nit here. In some cases the `static int` (or whatever) is on a line by itself, in others it is on the same line as the function. I prefer the latter (which is also what is generally used in PROJ) but at least it should be consistent across the file.

Comment (schwehr) [reply src/PJ_isea.c]: Done.  I was trying not to worry to much about formatting without the project having an official style (or better yet a clang-format definition I can just run before committing).

Comment (kbevers) [reply src/PJ_isea.c]: I know. Thanks for doing the touch up. About the style guide... we'll get there eventually :-)

**Pull request merged**